packages/playwright/tests/axe-playwright.spec.ts

```diff
@@ -459,48 +459,38 @@ describe('@axe-core/playwright', () => {
 
     it('with include using an array of strings', async () => {
       await page.goto(`${addr}/context.html`);
-      const expected = ['.selector-one', '.selector-two', '.selector-three'];
-
-      const axeSource = `
-      window.axe = {
-        configure(){},
-          run({ include }){
-            return Promise.resolve({ include })
-          }
-      }
-    `;
-      const results = new AxeBuilder({ page, axeSource: axeSource }).include([
-        '.selector-one',
-        '.selector-two',
-        '.selector-three'
-      ]);
+      const results = await new AxeBuilder({ page })
+        .include(['.include', '.include2'])
+        .analyze();
 
-      const { include: actual } = (await results.analyze()) as any;
+      const flattenTarget = flatPassesTargets(results);
 
-      assert.deepEqual(actual[0], expected);
+      assert.strictEqual(flattenTarget[0], '.include');
+      assert.strictEqual(flattenTarget[1], '.include2');
     });
 
     it('with exclude using an array of strings', async () => {
       await page.goto(`${addr}/context.html`);
-      const expected = ['.selector-one', '.selector-two', '.selector-three'];
-
-      const axeSource = `
-      window.axe = {
-        configure(){},
-          run({ exclude }){
-            return Promise.resolve({ exclude })
-          }
-      }
-    `;
-      const results = new AxeBuilder({ page, axeSource: axeSource }).exclude([
-        '.selector-one',
-        '.selector-two',
-        '.selector-three'
-      ]);
+      const results = await new AxeBuilder({ page })
+        .exclude(['.selector-one', '.selector-two', '.selector-three'])
+        .analyze();
+      const flattenTarget = flatPassesTargets(results);
 
-      const { exclude: actual } = (await results.analyze()) as any;
+      assert.notInclude(flattenTarget, '.selector-one');
+      assert.notInclude(flattenTarget, '.selector-two');
+    });
+    it('with chaining include/exclude using an array of strings', async () => {
+      await page.goto(`${addr}/context.html`);
+      const results = await new AxeBuilder({ page })
+        .include(['.include', '.include2'])
+        .exclude(['.exclude', '.exclude2'])
+        .analyze();
+      const flattenTarget = flatPassesTargets(results);
 
-      assert.deepEqual(actual[0], expected);
```

Comment:
We would need to keep this exclude test, it checks that the exclude selectors are getting passed to axe-core